src/app/components/data-entry-grid/index.html

```diff
@@ -0,0 +1,232 @@
+      <sky-docs-anatomy imagePath="~/assets/img/guidelines/data-grid/anatomy-view.png">
+        <sky-docs-anatomy-item anatomyNumber="1">Header row</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="2">Unselected row</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Multiselect checkbox</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Selected row</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Options">
+		<h3>Disabled cells</h3>
+      <sky-docs-visual-example>
+				<sky-docs-visual-example-text>
+					<p>You can disable columns of cells within the grid if users should not edit that data.</p>
+				</sky-docs-visual-example-text>
+				<sky-docs-visual-example-image>
+				  <img src="~/assets/img/guidelines/data-grid/option-disable.png" />
+				</sky-docs-visual-example-image>
+			</sky-docs-visual-example>
+    <h3>Multiselect</h3>
+			<sky-docs-visual-example>
+				<sky-docs-visual-example-text>
+					<p>You can enable multiselect if users need to select multiple rows in the grid to take action on.</p>
+				</sky-docs-visual-example-text>
+				<sky-docs-visual-example-image>
+				  <img src="~/assets/img/guidelines/data-grid/option-multiselect.png" />
+				</sky-docs-visual-example-image>
+			</sky-docs-visual-example>
+	</sky-demo-page-content>
+
+  <sky-demo-page-content sectionHeading="Related information">
+		<h3>Components</h3>
+			<ul>
+				<li><a routerLink="/components/grid">Grid</a></li>
+			</ul>
+		<h3>Guidelines</h3>
+			<ul>
+				<li><a routerLink="/design/guidelines/form-design">Form design</a></li>
+			</ul>
+	</sky-demo-page-content>
+
+    </sky-tab>
+
+    <sky-tab tabHeading="Development">
+      <p>
+        The SKY UX ag-Grid module provides styles, a service for default gridOptions, and SKY UX components for cell rendering and editing in ag-Grid. If you have never used ag-Grid before, <a href="https://www.ag-grid.com/angular-getting-started/">the Angular getting started</a> guide covers the basics. For more in depth information about ag-Grid and its capabilities, please reference <a href="https://www.ag-grid.com/documentation-main/documentation.php">the ag-Grid documentation.</a>
+      </p>
+
+      <sky-demo-page-module-info
+        skyuxModule="SkyAgGridModule"
+        packageName="@skyux/ag-grid"
+        githubRepo="skyux-ag-grid"
+      >
+      </sky-demo-page-module-info>
+
+      <sky-demo-page-properties
+      sectionHeading="SkyAgGridService methods"
+      >
+        <p>
+          <sky-code>SkyAgGridService</sky-code> is a service that provides methods to get ag-Grid <sky-code>gridOptions.</sky-code> The <sky-code>gridOptions</sky-code> can be overridden, and include registered SKY UX column types. Use the service when you are using ag-Grid to ensure your grid matches the standard SKY UX functionality.
```

Comment:
In the first sentence, I'd delete "is a service that." Would it be true to say "`<sky-code>SkyAgGridService</sky-code>` is a service that provides methods to get ag-Grid `<sky-code>gridOptions</sky-code>` to ensure grids match SKY UX functionality"? If yes, then we can eliminate the last third sentence.